shared/menubar/index.js

```diff
@@ -187,8 +187,8 @@ class Menubar extends Component {
 
 export default connect(
   state => ({
-    username: state.config && state.config.status && state.config.status.user && state.config.status.user.username,
-    loggedIn: state.config && state.config.status && state.config.status.loggedIn,
+    username: state.config.username,
```

Comment:
Mm mm we should talk to Marco. I think this is cause it doesn't have the
store at all so it's all empty. Maybe that should be handled at a higher
level

On Wednesday, May 11, 2016, Max Goodman <notifications@github.com> wrote:

> In shared/menubar/index.js
> <https://github.com/keybase/client/pull/2811#discussion_r62947426>:
>
> > @@ -187,8 +187,8 @@ class Menubar extends Component {
> >
> >  export default connect(
> >    state => ({
> > -    username: state.config && state.config.status && state.config.status.user && state.config.status.user.username,
> > -    loggedIn: state.config && state.config.status && state.config.status.loggedIn,
> > +    username: state.config.username,
>
> @chrisnojima <https://github.com/chrisnojima> Hmm, this throws when I
> open the menu widget -- it seems that state.config isn't defined at the
> time that it opens. Is that what you expected? Should I dive into this
> further?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2811/files/8bd32bbe6f405eb610c54c9a398622e9c4eaa14d#r62947426>
>